src/utils/queryOps.js

```diff
@@ -136,11 +191,35 @@ extend_wrapper({
     return lodashFilter(list, func);
   },
 
+  exists: function exists(list, columns = null) {
+    // Expect a list of column names that must exist
+    let func = null;
+
+    if (columns == null) {
+      func = row => row != null;
+    } else {
+      const cols = toArray(columns);
+      func = (row) => {
+        for (const name of cols) {
+          const v = row[name];
+          if (v == null || Number.isNaN(v)) return false;
+        }
+        return true;
+      };
+    }
+    return lodashFilter(list, func);
+
+  },
+
+  append: function append(list, value) {
+    return [...list, value];
+  },
+
```

Comment:
I'm wondering the utility of putting so many of these simple methods and syntax into a class, rather than using stand-alone helpers or just doing them in place.

Reply:
You could also do this: 
```
const transformedData = myArray.where({bits});
const newData = [...existing, ...v];
```

Reply:
I see what you mean about chaining methods, but you can still use the built in javascript methods without abstracting them into a class method (like .reverse()).